clang/include/clang/CIR/Dialect/Builder/CIRBaseBuilder.h

```diff
@@ -327,4 +337,4 @@ class CIRBaseBuilderTy : public mlir::OpBuilder {
 
 } // namespace cir
 
-#endif
+#endif
```

Comment:
This indicates that you (or some tool you're using) removed the newline character at the end of the file. It's a good practice to configure your editor to automatically add a newline at the end if it's missing. If you use the VSCode editor, the text editor has a setting called "Insert Final Newline" that does this.